src/main/java/org/umssdiplo/automationv01/core/managepage/organizationalstructuremenu/OrganizationalStructureMenu.java

```diff
@@ -8,19 +8,25 @@
 import org.umssdiplo.automationv01.core.utils.CommonEvents;
 
 public class OrganizationalStructureMenu extends BasePage {
+
+    @FindBy(xpath = "//span[text() = 'Estructura Organizacional']")
+    private WebElement employeeMenu;
+
     @FindBy(id = "sub-menu-positions")
     private WebElement positionSubMenu;
 
-    @FindBy(id = "sub-menu-employees")
+    @FindBy(xpath = "//span[text() = 'Empleados']")
     private WebElement employeeSubMenu;
 
     public OrganizationalStructureMenu() {
-        CommonEvents.isVisible(positionSubMenu);
+        CommonEvents.isVisible(employeeMenu);
```

Comment:
why are you asking to be visible this menu?
Currently this constructor is checking the last element to be loaded which is "cargos" menu